packages/sdk/tests/new/Fuse/utils.spec.ts

```diff
@@ -0,0 +1,103 @@
+import proxyquire from "proxyquire";
+import { SinonSpy, SinonStub, spy, stub } from "sinon";
+
+import { expect } from "../globalTestHook";
+import { mkAddress } from "../helpers";
+
+describe("Fuse utils", () => {
+  let solidityKeccak256Spy: SinonSpy;
+  let keccak256Spy: SinonSpy;
+  let getCreate2AddressSpy: SinonSpy;
+  let ContractFactoryStub: SinonStub;
+  let ContractStub: SinonStub;
+  let AbiCoderStub: SinonStub;
+
+  let utilsFns;
+
+  beforeEach(() => {
+    utilsFns = proxyquire("../../../src/Fuse/utils", {
```

Comment:
I am using proxyquire here to proxy nested function. That was not possible to do with sinon.
Let me know if this is not ok

Reply:
https://github.com/Midas-Protocol/monorepo/blob/sdk-test/packages/sdk/tests/new/Fuse/index.spec.ts#L6

Did you see how this was done?